src/components/PostPreview/PostPreviewContent.tsx

```diff
@@ -0,0 +1,38 @@
+import './PostPreview.scss';
+
+import { Heading, Link, Skeleton, Text } from '@eleven-labs/design-system';
+import React from 'react';
+
+import { PostPreviewOptions } from './PostPreview';
+
+const PostPreviewContent = ({
+  isLoading,
+  hasMask,
+  title,
+  link,
+  isRelated,
+  excerpt,
+}: Partial<PostPreviewOptions>): React.ReactNode => {
```

Comment:
Créer une interface dédier pour ce composant
```suggestion
export const PostPreviewContent: React.FC<PostPreviewContentProps> = ({
  isLoading,
  hasMask,
  title,
  link,
  isRelated,
  excerpt,
}) => {
```